gui_agents/agents/Backend/LybicBackend.py

```
@@ -12,227 +19,335 @@
+            # 使用getattr以防属性名不同
+            self.sandbox_id = getattr(new_sandbox, 'id', "") or getattr(new_sandbox, 'sandbox_id', "")
+            if not self.sandbox_id:
+                raise RuntimeError(f"Failed to get sandbox ID from response: {new_sandbox}")
+            print(f"Created sandbox: {self.sandbox_id}")
+
+    def __del__(self):
+        """清理资源"""
+        try:
+            if hasattr(self, 'client'):
+                self.loop.run_until_complete(self.client.close())
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)